packages/vscode-extension/src/debug/teamsfxTaskProvider.ts

```diff
@@ -73,6 +74,13 @@ const customTasks = Object.freeze({
     presentationEcho: false,
     presentationshowReuseMessage: false,
   },
+  [TaskCommand.launchDesktopClient]: {
+    createTerminal: (d: vscode.TaskDefinition) =>
+      Promise.resolve(new LaunchDesktopClientTerminal(d)),
+    presentationReveal: vscode.TaskRevealKind.Silent,
```

Comment:
Double confirm - where to show the logs, output panel or terminal?
The `Silent, true, true` is to show logs in terminal (like dev tunnel task).
To totally hide terminal and show logs to output panel (like provision/deploy task), use `Never, false, false`.